### ISSUE TITLE
Subtitles appearing on the top of the screen

### ISSUE THREAD

**Issue by Vitorvlv** [jellyfin/jellyfin-android#210]
Hey, I'm having an issue with Jellyfin for Android where the subtitles are appearing on the top of the screen, barely visible.

I'll attach a screenshot showing it.

I'm running a custom rom with Android 10. 10.4.0 as my server and whatever the latest version of the app that is on the google play right now.
![Screenshot_20191013-041649](https://user-images.githubusercontent.com/26132870/66713793-9784b200-ed85-11e9-8738-1737ccb207b7.png)

**Comment by anthonylavado** [comment]
Could you tell us what version of the System Webview is?

**Comment by Vitorvlv** [comment]
Sure, I have here Android System Web view version 74.0.3729.186

**Comment by anthonylavado** [comment]
@Vitorvlv So it looks like we ran in to this issue before, with Chrome 74 (see https://github.com/jellyfin/jellyfin-web/issues/298).

It is possible to update your Android System WebView at all?

**Comment by Vitorvlv** [comment]
I had to download the Android System WebView from the play store for it to work, It wasn't showing me any updates. Custom rom quirks I guess. 

Subtitles are now showing fine, but now another issue showed up. I don't have a pause button. I just tested and it went away immediately after I paused the video for the first time. Is this a known thing already? Or should I open an issue for it?
![Screenshot_20191015-193627](https://user-images.githubusercontent.com/26132870/66875109-899c8000-ef83-11e9-9546-140b37b53be5.png)
